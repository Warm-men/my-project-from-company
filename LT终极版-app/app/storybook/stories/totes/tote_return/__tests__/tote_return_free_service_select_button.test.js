import React from 'react'
import renderer from 'react-test-renderer'
import SelectButton from '../tote_return_free_service_select_button'

describe('select button in tote return free service', () => {
  it('should render correctly when focus', () => {
    const tree = renderer
      .create(
        <SelectButton text={'全部留下'} type="focus" selectStatus="focus" />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  it('should render correctly when focus', () => {
    const tree = renderer
      .create(
        <SelectButton text={'全部留下'} type="focus" selectStatus="not-focus" />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
