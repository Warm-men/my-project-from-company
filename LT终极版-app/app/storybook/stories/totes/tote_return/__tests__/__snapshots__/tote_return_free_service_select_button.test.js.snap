// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`select button in tote return free service should render correctly when focus 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F2BE7D",
      "borderRadius": 2,
      "height": 32,
      "justifyContent": "center",
      "opacity": 1,
      "width": 104,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": 14,
      }
    }
  >
    全部留下
  </Text>
</View>
`;

exports[`select button in tote return free service should render correctly when focus 2`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "borderColor": "#CCCCCC",
      "borderRadius": 2,
      "borderWidth": 1,
      "height": 32,
      "justifyContent": "center",
      "opacity": 1,
      "width": 104,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#5E5E5E",
        "fontSize": 14,
      }
    }
  >
    全部留下
  </Text>
</View>
`;
